azurerm/resource_arm_kubernetes_cluster_test.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/hashicorp/terraform/helper/resource"
 	"github.com/hashicorp/terraform/terraform"
 	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/tf"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/validate"
 )
 
 func TestAzureRMKubernetesCluster_agentPoolName(t *testing.T) {
```

Comment:
Could we move this to the test file next to the function's implementation? Doesn't make much sense to be here

Reply:
Thanks for pointing it out. I've merged the tests cases into `TestKubernetesAgentPoolName` in `kubernetes_test.go`.